src/main/java/de/retest/util/ReflectionUtilities.java

```diff
@@ -22,7 +22,7 @@
 
 	public static List<Field> getAllFields( Class<?> clazz ) {
 		final List<Field> result = new ArrayList<>();
-		while ( !clazz.getName().equals( "java.lang.Object" ) ) {
+		while ( clazz.isAssignableFrom( Object.class ) ) {
```

Comment:
isAssignableFrom checks for Instance and superclass, thus you would also take into account `Object`